docs/DeveloperGuide.md

```diff
@@ -157,6 +157,23 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 ## **Implementation**
 
 ### Implemented features
+#### Contact features
+**SortContact feature**
+The `sort` command (class SortContactCommand) is a SortContact feature which allows a user to sort the current person list in alphabetical order permanently. Contacts that are being added to the list later will not be sorted and added to the end of the list.
```

Comment:
should it be able to go back to chronological order? else, it feels like this should be our default setting since there's no other option

Reply:
No apparently I tried to think of ways to do it but we could not because of the fact that UI get the Filtered List instead of the Sorted List, unless we implement the undo feature?